tool/tsh/common/tsh.go

```diff
@@ -1927,21 +1927,13 @@ func onLogin(cf *CLIConf) error {
 		return trace.Wrap(err)
 	}
 
-	// Show on-login alerts, all high severity alerts are shown by onStatus
-	// so can be excluded here, except when Hardware Key Touch is required
-	// which skips on-status alerts.
-	alertSeverityMax := types.AlertSeverity_MEDIUM
-	if tc.PrivateKeyPolicy == keys.PrivateKeyPolicyHardwareKeyTouch {
```

Comment:
Do we no longer need to treat hardware key touch as a special case?

Reply:
We're now always doing what we used to only do for hardware key touch, so it's effectively always the special case.